releases/latest/beta/Dockerfile.ubi.openjdk25

```diff
@@ -0,0 +1,157 @@
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-25-m2-jre-ubi9-minimal AS getRuntime
```

Comment:
- Let's use the naming convention used in kernel-slim and full for UBI Minimal-based Dockerfiles: `releases/latest/veta/Dockerfile.ubi9-minimal.openjdk25`
- Delete `releases/latest/beta/Dockerfile.ubi.openjdk21`

Reply:
@idlewis Thanks for renaming the beta Dockerfile. Please address the second item above when you get a chance.